### ISSUE TITLE
Metric discrepancy between osx and ubuntu

### ISSUE THREAD

**Issue by degzhaus** [macbre/phantomas#537]
Hello, I have been trying to understand what could be causing significant metric discrepancy between phantomas numbers when run from my local osx environment and phantomas numbers when run from an ubuntu docker container running on my local osx environment.

The following gist shows the discrepancy in the results as well as a full log of each run:
https://gist.github.com/degzhaus/dcf966a96a3764defaea

Both environments are running: phantomas v1.11.0

What could be causing such a large discrepancy? 

Thank you!

John

**Comment by macbre** [comment]
@degzhaus, there are `SSL handshake failed` errors reported:

* 13 when run in Ubuntu (only 13 domains are used to fetch the page)
* just 2 for osx (51 domains are used)

**Comment by gmetais** [comment]
I'm also seeing more SSL handshake errors on Ubuntu than on Mac. It's a PhantomJS problem.
I tried many things, such as updating the ca-certificates, without success.

Several open issues about this topic in PhantomJS, such as [this one](https://github.com/ariya/phantomjs/issues/12181).

**Comment by degzhaus** [comment]
` Vitallium committed a day ago` https://github.com/ariya/phantomjs/commit/54a54dc0687a4b93562285d4f632dcafe2c7bf4f

:+1:

**Comment by macbre** [comment]
Depends on #488

**Comment by macbre** [comment]
@degzhaus - #531 is now merged into devel branch. You can use PhantomJS 2.0.0 by settings an environment variable:

```
PHANTOMAS_ENGINE=webkit2 phantomas "http://example.com"
```

or passing --engine=webkit2 option.

Can you please try to reproduce your case using PhantomJS 2.0.0? Thanks :)
